src/components/Chip/Chip.tsx

```diff
@@ -18,85 +17,54 @@ export interface Props {
   moreInfoComponentShowOn?: DisplayMoreInfo;
   style?: React.CSSProperties;
   theme?: any;
-  onRemove?(event: React.FormEvent<HTMLElement>): void;
-  onClick?(event: React.FormEvent<HTMLElement>): void;
+  onRemove?(event: any): void;
+  onClick?(event: any): void;
   handleMoreInfo?(): void;
-  key?: number;
-  tabIndex?: number;
-  autoSuggestMethods?: IAutoSuggestMethods;
-  onfocus?(event: Event): void;
-  id?: number;
+  key?: any;
+  markedForDelete?: boolean;
+  children?: string;
```

Comment:
Sync up chip component with Documentation available in docs app. Both should match and if required update docs

Reply:
resolved